### DISCUSSION TITLE
Winter CMS - Blocks Theme

### DISCUSSION THREAD

**Discussion by damsfx** [wintercms/winter#139] [Show and tell]
Winter CMS theme that demonstrates the use of "repeater groups" in [custom page fields](https://github.com/wintercms/wn-pages-plugin/blob/main/docs/documentation.md#custom-page-fields) to assemble static pages with customizable building sections and blocks.  

[Winter CMS - Blocks Theme](https://github.com/Hounddd/wn-blocks-theme)

![](/Hounddd/wn-blocks-theme/blob/main/assets/images/blocks.jpg?raw=true)
Each section, depending on his type, can have one or more blocks.  
![](/Hounddd/wn-blocks-theme/blob/main/assets/images/blocks-menus.png?raw=true)

**Have fun with Winter CMS!**

**Comment by bennothommo** [comment]
Nice one @damsfx! :)

**Comment by ajaxStardust** [comment]
hey that seems cool. do you have any code samples to work with, or... 
i dunno.  i like to reference https://tachyons.io/components/ 
note the way one of lesser talents might scoff at such shoddy presentation, but it's really better that way. 
remember copying the view source into notepad. aww yeah. i do. haha. 
what the actual heck. 

i hate wordpress.

check this out.
https://youtu.be/6Aup8_JgBsY?si=W6Nmssu1pcqqy4wn

any editor that has a terminal with access to the buffer can launch a PHP preview (or any Server-side scripting language), as an HTTP URL, no weird port numbers, in your favorite Browser.  Maintaining your productivity flow in VSCode, Sublime Text, Zed, EmEditor, Notepad3, Notepad++ is easy once you get the hang of it.
